pkg/kubectl/kubectl_test.go

```diff
@@ -0,0 +1,51 @@
+package kubectl
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"path"
+	"testing"
+)
+
+func readFile(t *testing.T, path string) []byte {
+	dat, err := ioutil.ReadFile(path)
+	if err != nil {
+		t.Fatal(err)
+	}
+	return dat
+}
+
+func jsonToMap(t *testing.T, b []byte) map[string]interface{} {
+	obj := make(map[string]interface{})
+	err := json.Unmarshal(b, &obj)
+	if err != nil {
+		t.Fatal(err)
+	}
+	return obj
+}
+
+func TestEmptyStatus(t *testing.T) {
+	dir := "test-fixtures"
+	tcs := []string{
+		path.Join(dir, "pod-disruption-budget.json"),
+		path.Join(dir, "storage-class.json"),
+	}
+	for _, tc := range tcs {
```

Comment:
You could use [subtests](https://blog.golang.org/subtests) for this test cases. This has the advantage, that the test output would tell you exactly which test case failed.

See here for example: https://github.com/rebuy-de/aws-nuke/blob/3a52c40/cmd/config_test.go#L161

Reply:
Uh, nice! Added.